src/views/assets/Platform/PlatformList.vue

```diff
@@ -161,6 +161,15 @@ export default {
     } finally {
       this.loading = false
     }
+
+    const name = this.$route.query?.name
+    const platform = this.$route.query?.id
+
+    if (platform) {
+      this.$nextTick(() => {
+        this.$refs.genericListTable.onDetail({ row: { id: platform, name }})
+      })
+    }
   },
   updated() {
     this.changeMoreCreates()
```

Comment:
In the provided piece of code, there are no significant discrepancies to note for any irregularities, potential issues, or optimization suggestions. The code appears well-integrated and doesn't reveal any notable inconsistencies from previous versions.

However, it's important to consider that changes may not become available directly due to an embargo on the latest knowledge being made public. Therefore, users should be cautious when applying these changes without checking their local context.